soh/src/code/z_parameter.c

```diff
@@ -4971,7 +4972,7 @@ void Interface_Draw(PlayState* play) {
                         break;
                 }
             } else {
-                if (CVarGetInteger("gCosmetics.Consumable_GreenRupee.Changed", rupeeWalletColors)) {
+                if (CVarGetInteger("gCosmetics.Consumable_GreenRupee.Changed", 0)) {
```

Comment:
As a note, this is a bug caught by the checks I added in the new CVars system. The default value was a pointer, instead of 0 (false) that it was everywhere else.